app/test_main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.main import can_access_google_page
+from _pytest.monkeypatch import MonkeyPatch
+import pytest
+
+
+@pytest.mark.parametrize(
+    "internet,google,result",
+    [
+        (True, True, "Accessible"),
+        (True, False, "Not accessible"),
+        (False, True, "Not accessible"),
+    ]
+)
+def test_function_acces(monkeypatch: MonkeyPatch,
```

Comment:
After opening the parentheses, write each the arguments on a new line, including the first one, the closing bracket is also written on a new line

Reply:
do it